syft/pkg/cataloger/cpe.go

```diff
@@ -271,10 +305,42 @@ func candidateProductsForJava(p pkg.Package) []string {
 }
 
 func candidateVendorsForJava(p pkg.Package) []string {
-	return vendorsFromGroupIDs(groupIDsFromJavaPackage(p))
+	gidVendors := vendorsFromGroupIDs(groupIDsFromJavaPackage(p))
+	nameVendors := vendorsFromJavaManifestNames(p)
+	return strset.Union(gidVendors, nameVendors).List()
 }
 
-func vendorsFromGroupIDs(groupIDs []string) []string {
+func vendorsFromJavaManifestNames(p pkg.Package) *strset.Set {
+	vendors := strset.New()
+
+	metadata, ok := p.Metadata.(pkg.JavaMetadata)
+	if !ok {
+		return vendors
+	}
+
+	if metadata.Manifest == nil {
+		return vendors
+	}
+
+	for _, name := range javaManifestNameFields {
+		if value, exists := metadata.Manifest.Main[name]; exists {
+			if !startsWithDomain(value) {
```

Comment:
there is probably a better name for this (not domain, but TLD, for top level domain)... to be confirmed

Reply:
fixed in 1f396a69bdb91ba238685fb40c8e8a0c5a21345d

Reply:
fixed in 1f396a69bdb91ba238685fb40c8e8a0c5a21345d